addons/account/static/src/components/tax_totals/tax_totals.xml

```diff
@@ -24,16 +24,20 @@
                     <t t-else="">
                         <span class="o_tax_group_edit" t-on-click.prevent="() => this.setState('edit')">
                             <i class="fa fa-pencil me-2"/>
-                            <span class="o_tax_group_amount_value">
-                                <t t-out="props.taxGroup.formatted_tax_group_amount"/>
-                            </span>
+                            <div class="o_field_monetary">
```

Comment:
I've not tested, but maybe the class could be added the parent td and there could be something like "direction: ltr" on the `o_tax_group_edit_input` so the text direction is still reversed by rtlcss if it needs to be

Reply:
I added the div because when I apply direction ltr on a span doesn't work as I want. so If we just add the css the to the class 'o_tax_group_amount_value' for example it won't show the desired output. let me know if this is what you meant